src/main/kotlin/com/openlattice/collections/mapstores/EntityTypeCollectionMapstore.kt

```diff
@@ -0,0 +1,72 @@
+package com.openlattice.collections.mapstores
+
+import com.dataloom.mappers.ObjectMappers
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.openlattice.collections.EntityTypeCollection
+import com.openlattice.hazelcast.HazelcastMap
+import com.openlattice.mapstores.TestDataFactory
+import com.openlattice.postgres.PostgresArrays
+import com.openlattice.postgres.PostgresTable
+import com.openlattice.postgres.ResultSetAdapters
+import com.openlattice.postgres.mapstores.AbstractBasePostgresMapstore
+import com.zaxxer.hikari.HikariDataSource
+import java.sql.PreparedStatement
+import java.sql.ResultSet
+import java.util.*
+
+private val mapper: ObjectMapper = ObjectMappers.getJsonMapper()
```

Comment:
Might be worth getting a new one instead of global one.

Reply:
Updated -- what's the reason for this change?